docs/docs/cookbook/cel.md

```diff
@@ -75,3 +78,27 @@ Cloud SDK installer and AI tools like claude code use osascript.
 
 Also if you're using osascript to do this legitimately this will break your
 usage.
+
+## Prevent users from enabling SSH (Santa 2025.8+)
+
+As called out in [loobins](https://www.loobins.io/binaries/systemsetup/) the
+systemsetup command can be used to enable SSH.
+
+To block this create a signing ID rule for `platform:com.apple.systemsetup` and
+attach the following CEL program:
+
+```clike
+args.join(" ").contains("-setremotelogin on")
+```
+
+## Prevent users from enabling Remote Apple Event (Santa 2025.8+)
+
+As called out in [loobins](https://www.loobins.io/binaries/systemsetup/) the
+systemsetup command can be used to enable Remote Apple Events.
+
+To block this create a signing ID rule for `platform:com.apple.systemsetup` and
+attach the following CEL program:
```

Comment:
I think this might be the first case of multiple cookbook rules targeting the same binary. It might be nice to show how to combine these 2 new rules together.